TemplateProject/Cartfile.resolved

```diff
@@ -0,0 +1,6 @@
+github "steipete/Aspects" "1.4.2"
```

Comment:
not just this one but the Cartfile only contains Argo ... but there is a lot in this file